src/framework/accordionItem/AccordionItem.spec.jsx

```diff
@@ -14,14 +13,20 @@ describe('AccordionItem', () => {
     };
 
     describe('isActive', () => {
-      it('it has the proper class when it is active', () => {
+      it('is true then the component has proper class', () => {
         const testCase = TestCaseFactory.create(AccordionItem, props);
-        expect(
-          testCase.find(
-            '.accordion__item__content--active'
-          ).length
-        )
-        .toEqual(1);
+        expect(testCase.find('.accordion__item__content--active').length)
+          .toEqual(1);
```

Comment:
Could also do `expect(testCase.first('.accordion__item__content--active')).beBeDefined()` and `.not.toBe...` for the second case.